docs/modules/administration/pages/installation.adoc

```diff
@@ -104,13 +104,15 @@ TIP: If you are on Linux, `oc cluster up` will do the trick.
 
 == Bitbucket Repositories
 
-On Bitbucket, there must be an `OPENDEVSTACK` project filled with the necessary repositories such as `ods-core`. To set them up, use:
+On your own Bitbucket instance, a new `OPENDEVSTACK` project will be created with the necessary repositories, such as `ods-core`. To set them up, use:
 
 [source,sh]
 ----
 make prepare-bitbucket-repos
 ----
 
+Notice that `prepare-bitbucket-repos` expects a running Atlassian suite. It will then create the OPENDEVSTACK project and the repos inside. It will *not* spin up a Bitbucket instance. You'll need to setup Bitbucket prior to running this. You can make use of either the Ansible playbooks in infrastructure-setup (however beware that this hasn't been updated in a while and might be buggy) or take inspiration from the ods-devenv setup, which uses containers to run Bitbucket.
```

Comment:
I would have thought the section above `== Infrastructure` already explains you need Atlassian? Maybe only the reference to ods-devenv should be added there? Or is the above not clear for you?

Reply:
For a user who is *completely new* to the ODS ecosystem like me, I found the "Atlassian" part of the "Infrastructure" section a bit short in the docs:

> At this stage you can setup the Atlassian tools if they do not exist yet, or modify them as needed.

So it was not 100% clear what I _really_ needed before proceeding with the ODS installation.

In the actual installation document itself, I could not find any reference to what the Atlassian suite is composed of (despite the fact it may be something trivial for the people who has been developing ODS from scratch). IMHO some literature specifying that Jira + Bitbucket + Confluence + Crowd installation is required prior to ODS installation would be great to have it clarified in the aforementioned document.

I'll leave it up to you to decide what you consider best and do count with me to improve any docs if necessary 😊

Reply:
I see. Maybe you can expand the docs in the Infrastructure part to include the information you missed. And additionally add a "caution" to say that past this point you need to have that running, or everything below will just fail.